frontend/components/DataSet/_styles.scss

```diff
@@ -17,6 +17,7 @@
     font-weight: $bold;
     display: flex;
     gap: $pad-xsmall; // For deprecated sandbox icons
+    white-space: nowrap;
```

Comment:
is this needed only for this feature? I think this should be added where needed and not set as the default behaviour for DataSet

Reply:
Ok, so I realized this was needed when there wasn't a tooltip on "Fleet-app maintained slug", since I flagged it to design to put a tooltip and the tooltip which doesn't have the bug. I'm going to remove it with a comment that it likely should be added in and tested if eventually we find a bug elsewhere. Here's the video of the bug without `white-space: nowrap` with a long `string` for `<dt>`:


https://github.com/user-attachments/assets/b216971c-b09a-450f-8c4b-977b112dd6a0

Reply:
just for clarity I dont mind making the fix, but I think that should be more local to the issue and not on the `DataSet` component as that will be applied everywhere and could lead to unforeseen issues. So in this specific case adding those styles to fix in the Software details modal.